install_dependencies.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
 set -x
-pip install -r requirements.txt --user --upgrade
-pip install -r requirements-dev.txt --user --upgrade
+pip install -r requirements.txt --upgrade
+pip install -r requirements-dev.txt --upgrade
```

Comment:
Removing the `--user` option from the `pip` command because it causes errors while installing packages in `virtualenv`. I'm sure whether it could cause trouble or not when there is no `virtualenv`.

Reply:
given that travis doesn't use virtualenv I'd prefer if you put this back

Reply:
given that travis doesn't use virtualenv I'd prefer if you put this back

Reply:
The errors were reported by travis-ci. So, it can't be changed back.

Reply:
I believe it must be okay for travis-ci environment. I'm not sure whether there are any other environments using `install_dependencies.sh`. Removing `--user` will cause those environments to install packages to the system-wide.